registry/quilt_server/views.py

```diff
@@ -1630,16 +1630,12 @@ def list_users_detailed():
 
     events = (
         db.session.query(Event.user, Event.type, sa.func.count(Event.type))
-        .filter(Event.type.in_([Event.Type.INSTALL, Event.Type.PREVIEW]))
         .group_by(Event.user, Event.type)
         )
 
-    event_results = defaultdict(lambda: defaultdict(int))
+    event_results = defaultdict(int)
```

Comment:
I guess you don't need `defaultdict` anymore... Just use the dictionary comprehension syntax, like I showed earlier?

(Or did you want to avoid `.get(...)` later? Not really worth it, I'd say - especially given that `package_counts` is a regular dict.)

Reply:
I think it reads better with defaultdict than with the comprehension